clarity/src/vm/representations.rs

```diff
@@ -47,7 +47,7 @@ lazy_static! {
         *STANDARD_PRINCIPAL_REGEX_STRING, *CONTRACT_PRINCIPAL_REGEX_STRING
     );
     pub static ref CLARITY_NAME_REGEX_STRING: String =
-        "^[a-zA-Z]([a-zA-Z0-9]|[-_!?+<>=/*])*$|^[-+=/*]$|^[<>]=?$".into();
+        "^[a-zA-Z]([a-zA-Z0-9]|[-_!?+<>=/*])*$|^[-+=/*^~&|]$|^[<][=<]?$|^[>][=>]?$".into();
```

Comment:
`^[<][=<]?$|^[>][=>]?$` This can be simplified to: `^<[=<]?$|^>[=>]?$`.

Reply:
Good catch, done!  But, please sync with @jcnelson (and any other relevant persons) on the changing of `CLARITY_NAME_REGEX_STRING` based on his feedback above. I don't feel that I have the knowledge to contribute to the discussion of whether or not this change can be considered safe or not.